### ISSUE TITLE
Support multiple sticky comments OR get message from file

### ISSUE THREAD

**Issue by yelizariev** [marocchino/sticky-pull-request-comment#51]
I'd need to make a workflow where multiple messages are posted depending on some conditions. For example I want to post message X when CONDITION_X is true and Y when CONDITION_Y is true.

There are two solutions for this:

1. allow multiple sticky comments via custom HEADER:
```
    - uses: marocchino/sticky-pull-request-comment@v1.2.1
      with:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        if: CONDITION_X
        id: HEADER_X
        message: |

          Message X
    - uses: marocchino/sticky-pull-request-comment@v1.2.1
      with:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        if: CONDITION_Y
        id: HEADER_Y
        message: |

          Message Y
```

2. Read message from file
```
    - run: |
         CONDITION_X && echo "X" >> message.txt
         CONDITION_Y && echo "Y" >> message.txt
    - uses: marocchino/sticky-pull-request-comment@v1.2.1
      with:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        message_file: message.txt
```

I'd prefer option 2. @marocchino what do you think?

**Comment by marocchino** [comment]
In this issue, the first approach seems to be good.
Now I'm doing that with a hardcoded header value like this.
https://github.com/marocchino/sticky-pull-request-comment/blob/9550af3011c31590c3be2a1e634b9cecdebc9610/src/comment.ts#L1

It is possible to pass a header value as an argument and add it to the header.

We can choose option 2. In that case, the title of this issue should probably be "enter message from text file."

**Comment by marocchino** [comment]
And you can combine multiple outputs like this:

```yaml
- run: echo ::set-output name=action_fruit::strawberry
  id: fruit1
- run: echo ::set-output name=action_fruit::watermelon
  id: fruit2
- uses: marocchino/sticky-pull-request-comment@v1
  with:
    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    message: |   
      ${{ steps.fruit1.outputs.action_fruit }}
      ${{ steps.fruit2.outputs.action_fruit }}
```

**Comment by marocchino** [comment]
it released as v1.3.0.
enjoy it. :)

**Comment by yelizariev** [comment]
Wow, that was extremely fast :+1:
